test/jit/test_monkeytype.py

```diff
@@ -0,0 +1,13 @@
+import torch
```

Comment:
`monkeytype` is another library, the feature's name is PDT, please rename file.

Reply:
Agreed and fixed